docs/api-docs/provider-apis/tax/overview.mdx

```diff
@@ -143,6 +143,16 @@ BigCommerce sends requests for tax estimates on product item, shipping, and hand
 
 [View the Estimate Taxes API reference](/docs/rest-contracts/tax#estimate-taxes).
 
+### Performance
+
+Merchants expect fast tax estimate responses to facilitate a streamlined checkout experience for their shoppers, hence performance is an essential consideration.
+
+To protect the shopper checkout experience from overly slow tax responses, tax estimate requests will timeout after a set period of time. The default threshold of four seconds is intended to support a wide variety of use cases (e.g. large tax estimate requests).
+
+<Callout type="info">
+  You may request a lower timeout threshold to provide a better service to merchants.
+</Callout>
```

Comment:
An important highlight is that tax providers can request a _lower_ timeout than the default four second timeout. 

We have intentionally not highlighted that they can (and have, in the past) request a _higher_ timeout (up to ten seconds maximum at this time) since we do want to discourage this.

Reply:
My fear is that making this a field on provider registration will lead to more support chatter, specifically around _increasing_ this timeout value - although you could potentially apply the same thinking with even introducing this section.

I could see this as something we do once we achieve self-service tax provider registration (e.g. must be between 1 to 4 seconds - even if we've allowed for exceptions in some cases historically).